k8s/aat/common/idam/idam-api.yaml

```diff
@@ -22,6 +22,24 @@ spec:
       replicas: 1
       ingressHost: idam-api-aat.service.core-compute-aat.internal
       aadIdentityName: idam
+      keyVaults:
+        "idam-idam-aat":
+          resourceGroup: idam-idam-aat
+          excludeEnvironmentSuffix: true
+          secretRef: "kvcreds"
+          secrets:
+            - web-admin-client-secret
+            - notify-api-key
+            - am-password
+            - idm-password
+            - pin-user-pass
+            - event-bus-access-key
```

Comment:
These values are dynamically added to the vault by shared-infrastructure.
idam-api cannot share postgres db between aat and aat2

Reply:
A shared idam-api is a requirement. Without it people that started registration in IDAM 1.5.1. won’t be able to complete their registrations in IDAM 2.0 successfully.